lib/scenic/command_recorder/statement_arguments.rb

```diff
@@ -36,15 +36,25 @@ def options
         @options ||= @args[1] || {}
       end
 
+      def keyword_hash(hash)
+        if Hash.respond_to? :ruby2_keywords_hash
```

Comment:
@JoshCheek's original version conditionally defined this method, which means we don't have to call `respond_to?` every time call `keyword_hash` but `def`ing the method once is more aesthetically pleasing and overall less complex. Given that there is no performance concern with migrations, I think this is the right tradeoff.